aea/cli/registry/utils.py

```diff
@@ -24,7 +24,6 @@
 from json.decoder import JSONDecodeError
 
 import click
-
```

Comment:
@DavidMinarsch this was not correct. PEP8 does not specify spaces between imports of the same groups... The new approach is correct.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I guess there could be a way to introduce those empty lines... but not sure how much it makes sense.

Reply:
Ok let's bunch all third party in one group then